xml/ay_bigfile.xml

```diff
@@ -4712,6 +4712,99 @@ openssl x509 -noout -fingerprint -sha256
    </informaltable>
   </sect2>
 
+  <sect2 xml:id="ay.multidevice_btrfs">
+    <title>Multi-device Btrfs Configuration</title>
+
+    <para>
+      A <emphasis>Btrfs file system</emphasis> can be created on top of many devices, offering similar
+      features to a RAID device. With &ay;, a <emphasis>multi-device Btrfs</emphasis> can be configured
+      by specifying a drive with the <literal>CT_BTRFS</literal> type. The <literal>device</literal>
+      property is used as an arbitrary name to identify each <emphasis>multi-device Btrfs</emphasis>.
+    </para>
+
+    <para>
+      As with RAID, you need to create all block devices first (e.g., partitions, LVM logical volumes,
+      etc.) and assign them to the <emphasis>Btrfs file system</emphasis> you want to create over such
+      block devices.
+    </para>
+
+    <para>
+      The following example shows a simple <emphasis>multi-device Btrfs</emphasis> configuration:
+    </para>
+
+   <example>
+    <title>Multi-device Btrfs configuration</title>
+      <screen>
```

Comment:
Comment re the [screen] tag:

Screen indents the resulting output by how much the screen tag itself is indented in the source code. So, we avoid indenting sections within [screen]...[/screen] -- the opening tag should be right at the left margin, no indent at all, and then the first line of the content follows it immediately, without a newline.

The indent within the tags is only indented by as much as you want it to appear, regardless of the indentation levels of the XML source code.

Then the closing [/screen] tag goes right on the end of the last line of the output, again, without a CR/newline. This avoids extra blank lines before and after the output in the final file.